CONTRIBUTING.md

```diff
@@ -42,6 +42,12 @@ cargo make tests
 
 `cargo make tests` will automatically download Geckodriver to a temporary location if it isn't in the PATH.
 
+Because Geckodriver looks for `firefox` in the path, if you use
+FireFox Developer Edition, you may get an error, because Developer Editions
+binary is called `firefox-developer-edition` on some Linux distributions.
```

Comment:
It's probably not just Linux. Especially on Mac the situation is likely the same.
Let's just remove that clause.
```suggestion
binary is called `firefox-developer-edition`.
```